packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +75,24 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.srcdoc) {
+      if (frameOptions.iframeAttrs == null) {
```

Comment:
to be more specific
```suggestion
      if (frameOptions.iframeAttrs === null) {
```

Reply:
https://github.com/cerner/terra-framework/pull/1777/commits/5ceeb6dddfd25c257fde0c83c7fee19477b4a497